qemu/tests/virtio_fs_migration_on_error.py

```diff
@@ -0,0 +1,220 @@
+import os
+import re
+
+from virttest import error_context, utils_disk, utils_misc, utils_test, virt_vm
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Basic migration test with diffirent cache modes over localfs
+    Steps:
+        1. Create a shared directory on the host and write a file
+        2. Run the virtiofsd daemon on the host with different cache modes
+        3. Boot the source guest with the virtiofs device in step1
+        4. Mount the virtiofs targets inside the guest
+        5. Create a different directory on the host and write a file
+        6. Run the virtiofsd daemon to share the directory in step5
+        7. Boot the target guest with the virtiofs device in step5
+        8. Do migration from the source guest to the target guest
+        9. No error occurs, the virtiofs is mounted automatically and
+           the file content keeps the same on the target guest
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def create_service(session):
+        if os_type == "windows":
+            error_context.context("Create virtiofs service in guest.", test.log.info)
+
+            driver_name = params["driver_name"]
+
+            session = utils_test.qemu.windrv_check_running_verifier(
+                session, vm, test, driver_name
+            )
+            viofs_svc_name = params["viofs_svc_name"]
+            virtio_fs_utils.create_viofs_service(
+                test, params, session, service=viofs_svc_name
+            )
+        return session
+
+    def delete_service():
+        if os_type == "windows":
+            error_context.context("Delete virtiofs service in guest.", test.log.info)
+            session = vm.wait_for_login()
+            virtio_fs_utils.delete_viofs_serivce(test, params, session)
+            session.close()
+
+    def start_service(session):
+        def start_multifs_instance(fs_tag, fs_target, fs_volume_label):
```

Comment:
No multifs in this case.

Reply:
Good catch! The reason why I handle all of these in the 'multifs' way is that multifs is a test matrix and currently talked with Tingting, we can just cover this scenario in one test case, maybe in future, we will cover it from more test cases, so in the automation code, I considered this situation that it should be easy to extend to multifs later

Reply:
@zhencliu Thanks for your explanation, I prefer to remove them at the moment. 1) Not sure when will enable multifs. 2) If there are something update or bug fix for this part,  it would take resources to maintain them in more than one py files.

Reply:
I am OK with that, but maybe it might be better to confirm this with Tingting, hi @hellohellenmao , what's your opinion please?